DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/LookUpTable.h

```diff
@@ -0,0 +1,101 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+//
+//file RawEventData.h class  for RAW data format
+// Alla.Maevskaya
+//  simple look-up table just to feed digits 2 raw procedure.
+//Will be really set after module/electronics connections
+//
+#ifndef ALICEO2_FT0_LOOKUPTABLE_H_
+#define ALICEO2_FT0_LOOKUPTABLE_H_
+////////////////////////////////////////////////
+// Look Up Table FT0
+//////////////////////////////////////////////
+
+#include <Rtypes.h>
+#include <cassert>
+#include <iostream>
+#include <tuple>
+namespace o2
+{
+namespace ft0
+{
+struct Topo {
+  int mPM = 0;
+  int mMCP = 0;
+};
+
+inline bool operator<(Topo const& a, Topo const& b)
+{
+  return std::tie(a.mPM, a.mMCP) < std::tie(b.mPM, a.mMCP);
```

Comment:
``tie`` creates a tuple of references, why is it used here? Should this be
``return (a.mPM<b.mPM && a.mMCP<b.mMCP)``?